doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -69,6 +69,8 @@ The following list summarizes the most important changes inherited from the upst
 Thread
 ------
 
+* Enabled Thread 1.3 features by default.
+* Removed support for Thread Border Router as part of the Master library feature set.
 * Fixed a bug in which a Minimal Thread Device was not able to handle Address Error Notification messages.
 * Updated the values in the memory requirement tables in :ref:`thread_ot_memory` after the update to the :ref:`nrfxlib:ot_libs` in nrfxlib.
```

Comment:
These belong to the previous release, don't they?
```suggestion

```

Reply:
You are right.

Reply:
Agreed to remove on a separate PR.